web/cypress/support/commands.js

```diff
@@ -65,6 +65,7 @@ Cypress.Commands.add('configurePlugins', (cases) => {
           });
 
         // NOTE: wait for the Drawer to appear on the DOM
+        cy.get({ timeout });
```

Comment:
get what?

Reply:
![image](https://user-images.githubusercontent.com/72343596/105630517-b4fb6380-5e84-11eb-94d7-0a8537e670a8.png)
We can use timeout to replace those useless wait